lib/medusa/adapter/rabbitmq.ex

```diff
@@ -2,6 +2,7 @@ defmodule Medusa.Adapter.RabbitMQ do
   @moduledoc false
   @behaviour Medusa.Adapter
   use Connection
+  use Medusa.Logger, :publishing
```

Comment:
not sure that this needs to use macros, as types of logging could be passed in as argument. Is there particular reason to use this?